hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/metrics/SummedDistributedEphemeralCounter.java

```diff
@@ -0,0 +1,47 @@
+package pl.allegro.tech.hermes.management.infrastructure.metrics;
+
+import com.google.common.primitives.Longs;
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.zookeeper.KeeperException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+import static org.apache.commons.lang.exception.ExceptionUtils.getRootCauseMessage;
+
+public class SummedDistributedEphemeralCounter {
+
+    private static final Logger logger = LoggerFactory.getLogger(SummedDistributedEphemeralCounter.class);
+
+    private final List<CuratorFramework> curatorClients;
+
+    public SummedDistributedEphemeralCounter(List<CuratorFramework> curatorClients) {
+        this.curatorClients = curatorClients;
```

Comment:
What do you think about adding some immutability here?

Reply:
What exactly do you mean?

Reply:
I think it's not necessary.